app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Category;
+use Illuminate\Http\Request;
+use App\Http\Requests\CategoryRequest;
+
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        return Category::paginate(10);
```

Comment:
Could you update the pagination based on the requirement from the ticket acceptance criteria: "The default per-page option is 20. We support options for 50 and 100 categories per page".

Also, the response body data in REST API calls is usually structured in JSON, so can you return json response in each function